src/python/grpcio/grpc/_local_credentials.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 The gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""gRPC's local credential API."""
+
+import enum
+import grpc
+from grpc._cython import cygrpc
+
+
+@enum.unique
+class LocalConnectType(enum.Enum):
```

Comment:
Feels like `LocalConnectionType` reads better alongside the rest of Python API.

Reply:
This API is exposed internally. Change the naming might need another large scale change.